root/release_group/edit_form.tt

```diff
@@ -26,6 +26,72 @@
     [%- enter_edit() -%]
 
   </div>
+
+  <div class="documentation">
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is usually the title from the earliest official release in the release group.',
+             { doc => { href => doc_link('Release_Group#Title'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Release_Group#Title'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="artist-bubble">
+      <p>
+        [% l('The {doc|artist} should usually match the earliest official release in the release group.',
+             { doc => { href => doc_link('Release_Group#Artist'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Release_Group#Artist'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field helps users distinguish between similarly-named release groups by the same artist,
+              such as a single and a full album with the same name.',
```

Comment:
Single vs album is literally a case where we say not to add a disambiguation in the docs :p The actual use case is often artists with several self-titled albums.

Reply:
Hah, true. How about this?

> The {doc|disambiguation} field helps users distinguish between similarly-named release groups, such as multiple self-titled albums by the same artist.
>
> Please leave it blank if other information such as artist or type (album, single, etc.) already distinguishes between the release groups.